src/main/kotlin/org/axonframework/intellij/ide/plugin/api/AxonAnnotation.kt

```diff
@@ -0,0 +1,27 @@
+package org.axonframework.intellij.ide.plugin.api
+
+import com.intellij.psi.PsiField
+import org.jetbrains.uast.UField
+
+/**
+ * Contains used Axon annotations during analysis of the source code with their fully qualified name.
+ */
+enum class AxonAnnotation(val annotationName: String) {
+    COMMAND_HANDLER("org.axonframework.commandhandling.CommandHandler"),
+    EVENT_HANDLER("org.axonframework.eventhandling.EventHandler"),
+    EVENT_SOURCING_HANDLER("org.axonframework.eventsourcing.EventSourcingHandler"),
+    QUERY_HANDLER("org.axonframework.queryhandling.QueryHandler"),
+    COMMAND_HANDLER_INTERCEPTOR("org.axonframework.modelling.command.CommandHandlerInterceptor"),
+    SAGA_EVENT_HANDLER("org.axonframework.modelling.saga.SagaEventHandler"),
+
+    AGGREGATE_ROOT("org.axonframework.modelling.command.AggregateRoot"),
+    AGGREGATE_IDENTIFIER("org.axonframework.modelling.command.AggregateIdentifier"),
+    TARGET_AGGREGATE_IDENTIFIER("org.axonframework.modelling.command.TargetAggregateIdentifier"),
```

Comment:
I would add `ROUTING_KEY` next to `TARGET_AGGREGATE_IDENTIFIER`, as the `@RoutingKey` serves the purpose of routing commands consistently in absence of the `@TargetAggregateIdentifier`.

Reply:
Since I am removing the annotator for command messages I'm removing this entry as well. But it's good to know `@TargetAggregateIdentifier` is meta-annotated. Did not know that!